dgraph/cmd/alpha/http_test.go

```diff
@@ -985,3 +986,54 @@ func TestQueryBackwardCompatibleWithGraphqlPlusMinusHeader(t *testing.T) {
 	require.NoError(t, err)
 	require.Equal(t, "2", resp.Header.Get(x.DgraphCostHeader))
 }
+
+func TestQueryRdfData(t *testing.T) {
+	require.NoError(t, dropAll())
+	require.NoError(t, alterSchema(`name: string @index(term) .`))
+
+	// empty RDF response
+	q1 := `
+	{
+	  balances(func: anyofterms(name, "Alice Bob")) {
+	    name
+	    balance
+	  }
+	}`
+	resp, _, err := queryWithTs(queryInp{body: q1, typ: "application/dql", respFmt: "rdf"})
+	require.NoError(t, err)
+	require.Equal(t, `{"data":""}`, resp)
+
+	m1 := `
+    {
+	  set {
+		_:alice <name> "Bob \"<the builder>\"" .
```

Comment:
Minor: This blank node is confusing. It says "Alice" but in the value there is Bob...

Reply:
Thanks for noticing Michel. Fixed it now.